config/locales/en.yml

```diff
@@ -330,5 +330,9 @@ en:
     summary_list:
       delete: Delete
       edit: Change
+    table:
+      filter_count: results for the term entered
```

Comment:
It's possible to find only one result, so the announced text is `1 results for the term entered`, which isn't perfect. I don't think this should be a blocking issue though, I'm happy for it to be addressed in a follow up PR.

Reply:
Yes that's true. Annoying but if we can return to it I guess that's the way to go.